test/cases/cookies_test.exs

```diff
@@ -1,22 +1,19 @@
 defmodule Distillery.Test.CookiesTest do
   use ExUnit.Case, async: true
-  use EQC.ExUnit
 
   @tag numtests: 100
-  property "generated cookies are always valid" do
-    forall cookie <- generated_cookie() do
-      is_valid_cookie(cookie)
-    end
+  test "generated cookies are always valid" do
+    Enum.each(1..100, fn _ ->
+      assert generated_cookie() |> is_valid_cookie()
+    end)
```

Comment:
MINOR: maybe we can use a random number instead of 1 and 100

Reply:
Enum.each here is just to run the test 100 times. We don't use the value